app/scripts/views/base.js

```diff
@@ -739,9 +739,9 @@ define(function (require, exports, module) {
     /**
      * Place the cursor at the given position within the input element
      *
-     * @param {selector | element} which
-     * @param {number} selectionStart - defaults to after the last character.
-     * @param {number} selectionEnd - defaults to selectionStart.
+     * @param {Selector | element} which
```

Comment:
Also, is `Selector` the correct type here or should it really be `String`? `Selector` feels more like it belongs in the description to me.

(and sorry, I know you haven't changed this bit)